align/pnr/build_pnr_model.py

```diff
@@ -148,14 +99,17 @@ def _ReadMap( path, mapname):
     d = pathlib.Path(path)
     p = re.compile( r'^(\S+)\s+(\S+)\s*$')
     tbl = {}
+    tbl2 = defaultdict(list)
     with (d / mapname).open( "rt") as fp:
         for line in fp:
             line = line.rstrip('\n')
             m = p.match(line)
             assert m
             k, v = m.groups()
             tbl[k] = str(d / v)
-    return tbl
+            tbl2[k].append( str(d / v))
```

Comment:
@854768750 Here tbl2 is the python version of the map from abstract name to list of concrete names.